python/mxnet/kvstore.py

```diff
@@ -11,30 +11,28 @@
 from . import optimizer as opt
 
 def _ctype_key_value(keys, vals):
-    """
-    Returns ctype arrays for the key-value args. For internal use.
-    """
+    names = []
     if isinstance(keys, int):
```

Comment:
well can I just test `isinstance(keys, (tuple, list))` and if it fails, cast single key to `str`?